### DISCUSSION TITLE
Github pages custom domain cannot be removed

### DISCUSSION THREAD

**Discussion by pulkitkhanna1** [community/community#30023] [Other Feature Feedback, Questions, & Ideas] [answered]
My domain validity has ended, and I want to remove the custom domain redirection, the remove button doesn't seem to work, Is there a way out?

**Comment by radar-blip** [comment, marked as answer]
Place your cursor in the text field and see if the remove button comes back. 
I dont  know if it will work, but cant hurt to try.

**Reply by radar-blip** [reply]
Maybe this is what you want: 
![unpublish](https://user-images.githubusercontent.com/39196767/185553109-211d5a23-b832-438d-8dd4-470462157db9.png)

Its under  Settings >> Pages

**Reply by pulkitkhanna1** [reply]
Thanks linuxlawson, issue id fixed, I deleted the cname from username.github.io repository

**Reply by radar-blip** [reply]
Awesome! Glad you got it fixed.
